src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -296,32 +304,27 @@ void expressionWithValidFakerObjectValidMethodInvalidArgs() {
     }
 
     @RepeatedTest(100)
-    void futureDateExpression() throws ParseException {
-        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS", Locale.ENGLISH);
-
-        Date now = new Date();
-        Date nowPlus10Days = new Date(now.getTime() + MILLIS_IN_A_DAY * 10);
-
-        Date date = dateFormat.parse(fakeValuesService.expression("#{date.future '10','TimeUnit.DAYS'}", faker, context));
-
-        assertThat(date.getTime()).isStrictlyBetween(now.getTime(), nowPlus10Days.getTime());
+    void futureDateExpression() {
+        LocalDateTime now = LocalDateTime.now(OffsetDateTime.now(ZoneId.systemDefault()).getOffset());
+        LocalDateTime nowPlus10Days = now.plusSeconds(SECONDS_IN_10_DAYs);
```

Comment:
why not do: 

`LocalDateTime nowPlus10Days = now.plusDays(10)` ?

Reply:
yea, that makes sense, probably another with hours should go same way
thanks